TheWall.ahk

```diff
@@ -224,16 +224,17 @@ ResumeInstance(pid) {
 SwitchInstance(idx)
 {
   if (idx <= instances) {
+    locked[idx] := false
     pid := PIDs[idx]
     if (instanceFreezing)
       ResumeInstance(pid)
-    WinMinimize, Fullscreen Projector
-    WinSet, AlwaysOnTop, On, ahk_pid %pid%
-    WinSet, AlwaysOnTop, Off, ahk_pid %pid%
     send {Numpad%idx% down}
     sleep, %obsDelay%
     send {Numpad%idx% up}
     WinMinimize, Fullscreen Projector
+    WinSet, AlwaysOnTop, On, ahk_pid %pid%
```

Comment:
This is sick but can you move the instance switching back above the obs switch :)